nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentManagerImpl.java

```diff
@@ -500,4 +497,21 @@ public boolean hasFeature(Feature feature) {
             throw new UnsupportedOperationException(feature.name());
         }
     }
+
+    /**
+     * @since 11.1
+     */
+    @Override
+    public DocumentRef getAncestorRef(CoreSession session, DocumentModel documentModel) {
+        return CoreInstance.doPrivileged(session, s -> {
+            if (!documentModel.hasSchema(COMMENT_SCHEMA)) {
```

Comment:
`s` doesn't seem to be used at all in the lambda, so `doPriviledged` is useless. Or is this a mistake somewhere else?